Makefile

```diff
@@ -14,6 +19,12 @@ lint: ## Check linting according to the flake8 configuration in setup.cfg
 mypy: ## Run typeckecking according to mypy configuration in setup.cfg
 	mypy .
 
+.PHONY: build
+build: ## Build the Python package (for debugging, building for release is done via a GitHub Action)
+	# (virtualenv installed due to Debian issue https://github.com/pypa/build/issues/224)
+	pip install --upgrade pip build virtualenv
+	python -m build --sdist --wheel
```

Comment:
Should this command be identical to the one in the GA to minimize dev confusion?